config-templates/ibm/ess/2.1.0/custom-parameters.json

```diff
@@ -0,0 +1,172 @@
+[
+	{
+		"description": "The mount path of the primary file system.",
+		"displayname": "Scale Host Path",
+		"name": "scale-host-path",
+		"required": "true",
+		"type": ""
```

Comment:
By default the value for `type` is `text`. 
Either remove `"type": ""` or explicitly set it to  `text`.

Reply:
removed "type":"".